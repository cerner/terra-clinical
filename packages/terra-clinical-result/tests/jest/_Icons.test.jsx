import React from 'react';
import Icons from '../../src/_Icons';

describe('Icons', () => {
  it('should render a default Icons', () => {
    const result = shallow(<Icons />);
    expect(result).toMatchSnapshot();
  });

  it('should render an Icons with isUnverified set', () => {
    const result = shallow(<Icons isUnverified />);
    expect(result).toMatchSnapshot();
  });

  it('should render an Icons with hasComment and isModified set', () => {
    const result = shallow(<Icons hasComment isModified />);
    expect(result).toMatchSnapshot();
  });
});
