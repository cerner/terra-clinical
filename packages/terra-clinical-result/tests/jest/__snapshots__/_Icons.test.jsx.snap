// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons should render a default Icons 1`] = `<Fragment />`;

exports[`Icons should render an Icons with hasComment and isModified set 1`] = `
<Fragment>
  <IconModified
    className="icon-modified"
    viewBox="0 0 48 48"
    xmlns="http://www.w3.org/2000/svg"
  />
  <IconComment
    className="icon-comment"
    data-name="Layer 1"
    isBidi={true}
    viewBox="0 0 48 48"
    xmlns="http://www.w3.org/2000/svg"
  />
</Fragment>
`;

exports[`Icons should render an Icons with isUnverified set 1`] = `
<IconDiamond
  className="icon-unverified"
  viewBox="0 0 48 48"
  xmlns="http://www.w3.org/2000/svg"
/>
`;
